src/main/groovy/io/saagie/plugin/dataops/tasks/service/VariableService.groovy

```diff
@@ -0,0 +1,25 @@
+package io.saagie.plugin.dataops.tasks.service
+
+import io.saagie.plugin.dataops.models.ListVariableContainer
+import org.codehaus.groovy.runtime.MethodClosure
+
+@Singleton
+class VariableService {
+	ArrayList<ListVariableContainer> listVariableContaines = []
+	
+	def getVariableList( variable, projectId, operation ) {
+		def found = listVariableContaines.find {
```

Comment:
found isn't accurate enough. What is found ?
please rename with foundVariable or foundSomething (idk what is found)